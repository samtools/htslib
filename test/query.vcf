##fileformat=VCFv4.1
##INFO=<ID=TEST,Number=1,Type=Integer,Description="Testing Tag">
##FORMAT=<ID=TT,Number=A,Type=Integer,Description="Testing Tag, with commas and \"escapes\" and escaped escapes combined with \\\"quotes\\\\\"">
##INFO=<ID=DP4,Number=4,Type=Integer,Description="# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases">
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
##FORMAT=<ID=GQ,Number=1,Type=Integer,Description="Genotype Quality">
##FORMAT=<ID=DP,Number=1,Type=Integer,Description="Read Depth">
##FORMAT=<ID=GL,Number=G,Type=Float,Description="Genotype Likelihood">
##FILTER=<ID=q10,Description="Quality below 10">
##FILTER=<ID=test,Description="Testing filter">
##contig=<ID=1,assembly=b37,length=249250621>
##contig=<ID=3,assembly=b37,length=198022430>
##contig=<ID=4,assembly=b37,length=191154276>
##reference=file:///lustre/scratch105/projects/g1k/ref/main_project/human_g1k_v37.fasta
##readme=AAAAAA
##readme=BBBBBB
##INFO=<ID=AC,Number=A,Type=Integer,Description="Allele count in genotypes">
##INFO=<ID=AN,Number=1,Type=Integer,Description="Total number of alleles in called genotypes">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	C	D
1	3062915	id3D	GTTT	G	48.7	q10	DP4=1,2,3,4;AN=4;AC=2	GT:GQ:DP:GL	0/1:409:35:-20,-5,-20	0/1:109:25:-10,-5,-20
1	3062915	idSNP	G	C,T	419	test	TEST=5;DP4=1,2,3,4;AN=4;AC=1,1	GT:TT:GQ:DP:GL	0/1:0,1:409:35:-20,-5,-20,-20,-5,-20	0/2:0,1:109:35:-10,-5,-20,-20,-5,-20
1	3062915	id2D	GTT	G	999	q10	DP4=1,2,3,4;AN=4;AC=2	GT:GQ:DP:GL	0/1:409:35:-20,-5,-20	0/1:109:25:-10,-5,-20
1	3106154	.	CAAA	C	72.6	PASS	AN=0;AC=0	GT:GQ:DP	.:245:32	./.:145:22
1	3106154	.	C	CT	459	PASS	AN=4;AC=2	GT:GQ:DP	0/1:245:32	0/1:145:22
1	3157410	.	G	T	46.7	q10	AN=4;AC=4	GT:GQ:DP	1/1:21:21	1/1:11:11
1	3162006	.	GAA	G	206	PASS	AN=4;AC=2	GT:GQ:DP	0/1:212:22	0/1:112:12
1	3177144	.	G	.	364	PASS	AN=4;AC=0	GT:GQ:DP	0/0:150:30	0/0:150:20
1	3184885	.	TAAAA	TA,T	8.42	PASS	AN=4;AC=2,2	GT:GQ:DP	1/2:22:20	1/2:12:10
2	3199812	.	G	GTT,GT	291	PASS	AN=4;AC=2,2	GT:GQ:DP	1/2:322:26	1/2:122:16
3	3212016	.	CTT	C,CT	52.5	PASS	AN=4;AC=2,2	GT:GQ:DP	1/2:91:26	1/2:11:16
4	3258448	.	TACACACAC	T	123	PASS	AN=4;AC=2	GT:GQ:DP	0/1:325:31	0/1:125:11
