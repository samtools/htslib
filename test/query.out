1	3062915	GTTT	G	1,2,3,4	4	0/1	GTTT/G	0/1	GTTT/G
1	3062915	G	C,T	1,2,3,4	4	0/1	G/C	0/2	G/T
1	3062915	GTT	G	1,2,3,4	4	0/1	GTT/G	0/1	GTT/G
1	3106154	CAAA	C	.	0	.	.	./.	./.
1	3106154	C	CT	.	4	0/1	C/CT	0/1	C/CT
1	3157410	G	T	.	4	1/1	T/T	1/1	T/T
1	3162006	GAA	G	.	4	0/1	GAA/G	0/1	GAA/G
1	3177144	G	.	.	4	0/0	G/G	0/0	G/G
1	3184885	TAAAA	TA,T	.	4	1/2	TA/T	1/2	TA/T
2	3199812	G	GTT,GT	.	4	1/2	GTT/GT	1/2	GTT/GT
3	3212016	CTT	C,CT	.	4	1/2	C/CT	1/2	C/CT
4	3258448	TACACACAC	T	.	4	0/1	TACACACAC/T	0/1	TACACACAC/T
